packages/eas-cli/src/commands/update/index.ts

```diff
@@ -134,6 +136,10 @@ export default class UpdatePublish extends EasCommand {
       default: 'all',
       required: false,
     }),
+    dev: Flags.boolean({
+      description: 'Publish an unminified dev bundle without stripping __DEV__ global',
```

Comment:
`__DEV__` is an implementation detail used under the hood to enable debugging features. best to not mention it in the high level description.
```suggestion
      description: 'Publish a development bundle',
```